src/TestInfrastructure/Helpers/MefTestHelpers.cs

```diff
@@ -108,7 +108,10 @@ public static Export CreateExport<T>(object exportInstance, IDictionary<string,
             for (int i = 0; i < requiredExports.Count(); i++)
             {
                 // Try importing when not all of the required exports are available -> exception
-                Exceptions.Expect<Exception>(() => TryCompose(importer, catalog, requiredExports.Take(i)));
+                Exceptions.Expect<Exception>(
+                    null,
+                    () => TryCompose(importer, catalog, requiredExports.Take(i)),
+                    checkDerived: true);
```

Comment:
Fixed a bug in the existing test helper: the method doesn't care which exception is thrown so it's ok for it to be a sub-class of _Exception_ (_checkDerived=true_. _allowDerivedExceptions_ would have been a better name).